From 0f8561648a63896c717c9428b713549c43f23799 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Thu, 28 May 2015 11:28:10 +0200
Subject: [PATCH 008/129] clk: shmobile: mstp: Document mandatory compatible
 fallback

The generic MSTP gate clocks compatible value is mandatory, as the
driver uses only this value for matching.  Document that this is a
fallback that must be present.

Also fix a typo (missing plural "s") in the compatible value.

Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
(cherry picked from commit 17df1fb2e502371b63ea1b2110584e8a712d6c20)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 Documentation/devicetree/bindings/clock/renesas,cpg-mstp-clocks.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/clock/renesas,cpg-mstp-clocks.txt b/Documentation/devicetree/bindings/clock/renesas,cpg-mstp-clocks.txt
index c95f7d2db6bd..16ed18155160 100644
--- a/Documentation/devicetree/bindings/clock/renesas,cpg-mstp-clocks.txt
+++ b/Documentation/devicetree/bindings/clock/renesas,cpg-mstp-clocks.txt
@@ -20,7 +20,7 @@ Required Properties:
     - "renesas,r8a7793-mstp-clocks" for R8A7793 (R-Car M2-N) MSTP gate clocks
     - "renesas,r8a7794-mstp-clocks" for R8A7794 (R-Car E2) MSTP gate clocks
     - "renesas,sh73a0-mstp-clocks" for SH73A0 (SH-MobileAG5) MSTP gate clocks
-    - "renesas,cpg-mstp-clock" for generic MSTP gate clocks
+    and "renesas,cpg-mstp-clocks" as a fallback.
   - reg: Base address and length of the I/O mapped registers used by the MSTP
     clocks. The first register is the clock control register and is mandatory.
     The second register is the clock status register and is optional when not
-- 
2.6.2

